components/StreamField/blocks/topics_block/TopicsBlock.module.scss

```
@@ -0,0 +1,40 @@
+.grid {
+    display: grid;
+
+    @include media-query(large) {
+        grid-template-columns: repeat(4, 1fr);
+        margin: 0 -10rem;
+    }
+}
+
+.card {
+    padding-bottom: 1rem;
+
+    @include media-query(large) {
+        padding: 0.5rem 1.75rem;
+    }
+
+    &__title {
+        @include font-size(m);
+        font-weight: 900;
+        line-height: 1.25;
+        margin: 0;
+    }
+
+    &__body {
+        @include font-size(s);
+        line-height: 1.5;
+        opacity: 0.7;
+        margin-top: 0.5rem;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        display: -webkit-box;
+        -webkit-line-clamp: 3;
+        line-clamp: 3;
+        -webkit-box-orient: vertical;
```

Cool! Does the SCSS precompiler not automatically add the vendor prefixes here?

```suggestion
        text-overflow: ellipsis;
        display: -webkit-box;
        -webkit-line-clamp: 3;
        -webkit-box-orient: vertical;
```

---

I'm not sure.

---

Looks like it doesn't, which is interesting... the only prefix to check is `line-clamp: 3` should automatically add `-webkit-line-clamp: 3`

---

interesting, lineclamp is supported mostly with `-webkit-line-clamp: 3` but not line-clamp.

https://caniuse.com/?search=line-clamp